webapp/components/ContentMenu.vue

```diff
@@ -175,6 +200,7 @@ export default {
         },
       })
     },
+    pinPost() {},
```

Comment:
This has no functionality and is superfluous!?

Reply:
good catch! thanks!